build/charts/Makefile

```diff
@@ -1,6 +1,10 @@
 USERID  := $(shell id -u)
 GRPID   := $(shell id -g)
 
+VERSION := $(shell head -n 1 ../../VERSION | cut -c 2-)
+
 .PHONY: helm-docs
 helm-docs:
 	docker run --rm --volume "$(CURDIR):/helm-docs" --user=$(USERID):$(GRPID) jnorwood/helm-docs:v1.7.0
+	sed -i.bak "s/0.0.0/$(VERSION)/g" theia/README.md # replace version placeholder
```

Comment:
Looks like you also need to change the version in `theia/README.md` to 0.0.0?

Reply:
I'm not sure whether you refer to the inconsistency between the `Chart.yaml` and `README.md`. As `README.md` is not expected to be modified when we package the chart but version number in `Chart.yaml` is taken care of by the package command. Thus the version number in `README.md` is replaced when it is generated to be consistent with the version number of the repo instead of the version number of the `Chart.yaml`.

Reply:
I see, thanks for the explanation!